zipline/assets/assets.py

```diff
@@ -350,7 +351,8 @@ def reload_symbol_maps(self):
         symbol maps.
         """
         # clear the lazyval caches, the next access will requery
-        for value in vars(type(self)).items():
+        for attr in dir(type(self)).items():
```

Comment:
`dir` returns a list of strings, so I don't think `dir(type(self)).items()` can be correct here.

Reply:
nothing even calls this function the test suite. nice!

Reply:
FWIW, the only call-site of this is removed on https://github.com/quantopian/zipline/pull/2192

Reply:
Okay, I removed the .items() because that is obviously wrong. I don't think we should invest any more time into this method if it is going away anyway.